app/views/application/_index_actions.html.slim

```diff
@@ -2,7 +2,7 @@ td= link_to t_action(:show), subject
 td= link_to t_action(:print), { controller: controller_name, action: :show, id: subject.id, print: 'true' }, target: '_blank'
 - if ['clients', 'volunteers'].include? controller_name
   td= link_to t_model(Journal), { controller: 'journals', action: 'index', "#{controller_name.singularize}_id": subject.id }
-td= link_to t_action(:edit), { controller: controller_name, action: :edit, id: subject.id }
-td
-  - if policy(subject.class).destroy?
-    = link_to t_action(:destroy), subject, confirm_deleting(subject)
+- if policy(subject.class).index?
+  td= link_to t_action(:edit), { controller: controller_name, action: :edit, id: subject.id }
```

Comment:
Hmm ok I see what you're doing now :)

One option to avoid overwriting `controller_name` would be to use another name for the variable like `target_controller`, and then have a line at the beginning of this view to fallback to `controller_name` so it still works in the cases where you're not overwriting it:

```slim
- target_controller ||= controller_name
```

But you can also leave it as it is now, I'm not sure my suggestion would really be better ;-)